__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data/index')
+const app = require('../app')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('/api/topics', () => {
+    test('GET: 200 responds with array of topic objects with slug and description properties', () => {
+        return request(app).get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            const { topics } = body
+            expect(topics.length).toBe(3)
```

Comment:
nice job remembering to put a length check in here, without it the forEach would pass even if it was an empty array!